File: tools/findLargeSum.py
def findLargeSum (a,b,c):
    suma = 0
    if a < b < c:
        suma = b + c
        return suma
    elif b < a < c:
        suma = a + c
        return suma
    elif c < a < b:
        suma = a + b
        return suma
    elif a < c < b:
        suma = b + c
        return suma
    elif c < b < a:
        suma = a + b
        return suma
    else:
        suma = b + c
        return suma

print(findLargeSum(2,3,1))
print(findLargeSum(31,51,42))

File: tools/findLargeSum.js
function findLargeSum (a,b,c) {
    var suma = 0;
    if (a < b < c) {
        suma = b + c;
        return suma
    } else if (b < a < c) {
        suma = a + c;
        return suma
    } else if (c < a < b) {
        suma = a + b;
        return suma
    } else if (a < c < b) {
        suma = c + b;
        return suma
    } else if (c < b < a) {
        suma = b + a;
        return suma
    } else {
        suma = b + c;
        return suma
    }
}

console.log(findLargeSum(2,3,1))
console.log(findLargeSum(31,52,42))